Topic0/practiceTopic0/Practice4/ConcreteBuilder1.java

```diff
@@ -0,0 +1,32 @@
+package practiceTopic0.Practice4;
+
+public class ConcreteBuilder1 implements BuilderInterface{
+	
+	private Connection connection;
+	
+	public ConcreteBuilder1() {
+			connection = new Connection();
+	}
+	
+	@Override
+	public void buildUser() {
+		connection.setUser("user1");
+	}
+	
+	@Override
+	public void buildPassword() {
```

Comment:
another thing noticed is that you are strong codding these parameters, why don't you use them as parameters? that way the user can use this class to create a connection with a different user,password etc...
public void withPassword(String password) {
connection.setPassword(password);
return this;
}